### PR TITLE
Use old ISO modal text

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

**Review by jtomasek — CHANGES_REQUESTED** [review]
Maybe we could keep the new UploadField usage + other tweaks and just put back the original Discovery flow description?

**Comment by andybraren** [comment]
+1 the new functionality would be great to keep; the boot order area just needs to be adjusted.

**Comment by rawagner** [comment]
OK, using mix of old and new :) 

Screenshot from 2020-08-26 19-57-11

File: src/components/clusterConfiguration/DiscoveryImageForm.tsx [hunk 1]
```diff
@@ -184,33 +153,18 @@ const DiscoveryImageForm: React.FC<DiscoveryImageFormProps> = ({
...
+                    Each host will need a valid IP address assigned by a DHCP server with DNS
+                    records that fully resolve.
                   </Text>
-                  <Text component={TextVariants.small}>
-                    Please configure each host's BIOS to boot in this sequence:
+                  <Text component={TextVariants.p}>
+                    The Discovery ISO should only be booted once per host. Either adjust the boot
+                    order in each host's BIOS to make it secondary after the first alphabetical
+                    disk, or select the ISO once manually. All other disks in the host will be wiped
+                    during the installation.
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/components/clusterConfiguration/DiscoveryImageForm.tsx]: sounds good!

**Review by eranco74 — APPROVED** [review]
/lgtm

**Review by jtomasek — APPROVED** [review]

**Pull request merged**